### PR TITLE
Meeting 94 notes

### PR DESCRIPTION
**Pull request by alita-moore**

### CODE REVIEW

File: All Core Devs Meetings/meeting 94.md [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+  
+## EIP Status
+EIP | Status
+  
+  
+## Decisions Made
+  
+Decision Item | Description
+--|--
+**94.1.1**: Discuss the possibility of launching a Prog POW on a test net next week in line with the Ben's compromise
```
Comment (poojaranjan) [inline All Core Devs Meetings/meeting 94.md:21]: Formatting

**94.1.1**| Discuss the possibility of launching a Prog POW on a test net next week in line with the Ben's compromise

File: All Core Devs Meetings/meeting 94.md [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+**94.1.2**: Move further discussion on precompile to ETH Magicians forum [here](https://ethereum-magicians.org/t/evm384-feedback-and-discussion/4533)
+  
+  
+  
+## Actions Required
+  
+Action Item | Description
+--|--
+**94.2.1**: Decide, by next meeting, whether to go with Axic's and update the sources that say we're going with EIP 2537
```
Comment (poojaranjan) [inline All Core Devs Meetings/meeting 94.md:30]: **94.2.1**| Decide, by next meeting, whether to go with Axic's and update the sources that say we're going with EIP 2537
**93.2.2**| Vitalik to start coordinating a writeup on the impact of the gas limit increase (what will it break)
**93.2.3**| revisit the discussion on EIP 384

File: All Core Devs Meetings/meeting 94.md [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+- [8. Migration from ACD Gitter to Eth R&D Discord](#8)
+- [9. Review previous decisions made and action items (if notes available)](#9)
+  
+---
+  
+# 1 Progress on EIP-1057
+  
+Video | [3:56](https://youtu.be/q6bIeSu7r9o?t=236)
+  
+**Hudson** - Hello and welcome to a theorem poor developer meeting No. 94, I'm your host, Hudson, and we're going to go to the first item on the agenda. Greg wants to give an update on the progress of EIP 10 57. So, Greg, if you want to admit and go through that real quick, that'd be great.
```
Comment (poojaranjan) [inline All Core Devs Meetings/meeting 94.md:63]: Hello and welcome to a Ethereum Core Developer meeting No. 94, I'm your host, Hudson, and we're going to go to the first item on the agenda. Greg wants to give an update on the progress of EIP 10 57. So, Greg, if you want to admit and go through that real quick, that'd be great.

File: All Core Devs Meetings/meeting 94.md [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+  
+**Hudson** - great. I do want to ask I'm not seeing in the decisions made section in the notes anything about us definitely spinning up testnets for prog pow or spinning up Ropstein or anything specific like that. Did we in your recollection, was there like a specific tests that we talked about spinning up for it, or was it Ropstein or what?
```
Comment (poojaranjan) [inline All Core Devs Meetings/meeting 94.md:67]: great. I do want to ask I'm not seeing in the decisions made a section in the notes anything about us definitely spinning up testnets for prog pow or spinning up Ropsten or anything specific like that. Did we in your recollection, was there like a specific tests that we talked about spinning up for it, or was it Ropsten or what?

File: All Core Devs Meetings/meeting 94.md [hunk 5]
```diff
@@ -0,0 +1,437 @@
...
+  
+**Hudson** - Oh, yeah. No, it's not I don't mean that we have to have consensus to spin up a test that we would for Ropstein, probably more so than just a random test that. Would you agree?
+  
+**James** - And something I have question I have for the groups we've talked earlier about how Rollston is at the point it's worth perhaps restarting. Is that still something that we think about or we still think is the case, or am I wrong that.
+  
+**Martin** - Yeah, before we go into that, yeah, I just want to answer Hudson's question. So the funny thing is, if we are to if we were to actually do a profile on Ropstein, is that I don't think it would cause any problems because I know first cinching would start with a Herbertson and would immediately see like what strange difficulties is this and it'll get rejected. So it's a lot easier than, for example, if we did the hard fork with an evm change, that's not immediately noticeable when you first think. So there's a difference there.
+  
+**Hudson** - Ok, cool, and then, James, you were asking everybody about rebooting Robstein or not rebooting Robstein but retiring Robstein, I guess you would say. Right?
```
Comment (poojaranjan) [inline All Core Devs Meetings/meeting 94.md:83]: Ok, cool, and then, James, you were asking everybody about rebooting Ropsten or not rebooting Robstein but retiring Ropsten, I guess you would say. Right?

File: All Core Devs Meetings/meeting 94.md [hunk 6]
```diff
@@ -0,0 +1,437 @@
...
+  
+**Vitalik** - I mean, I'd definitely be happy to like I would personally definitely be OK with that and then well, the thing that we want to keep in mind also in the long run is that in the long run, we would probably wants to move toward a regime where contract contract code has either a much larger limit or no limit because we have code virtualization and we will be charging for a chunk. So and and in that context, like basically having your code be in one contract versus having your code being split among a bunch of a bunch of different addresses would be it would be almost equivalent potentially. But I would definitely see in terms a personal with a slight increase in the max contract size, especially if it helps alleviate the effects to developers.
+  
+**Hudson** - Ok, Barton, did you have something? I think you're muted, Martin, if you're talking.
```
Comment (poojaranjan) [inline All Core Devs Meetings/meeting 94.md:211]: Ok, Martin, did you have something? I think you're muted, Martin, if you're talking.

File: All Core Devs Meetings/meeting 94.md [hunk 7]
```diff
@@ -0,0 +1,437 @@
...
+  
+**Martin** - Oh, yeah, I didn't know it was my time, so I just I mean, this proposed change, I'm all for it and it's technically very simple. And I guess the big problem is that this is something the community knows. It is going to break stuff up. 1884 broke some stuff. Most of it could be fixed. If we bump it from 800 to 2000, some things will be irrevocably broken. Yes, it's kind of like we have to just decide, yes, this is worth it for the future of Ethereum. And I think. You know, um I just want to get your thoughts on that. I think it's worth it, but still, 
+  
+**Vitalik** - In terms of the classes of things that would break like the mean, the biggest one is like twenty three hundred gas calls that are that do a storage look up, followed by a log, right. Do we know, like they also knew of a [?]?
+  
+***Martin** - I would say so, but one and also law at things which are linked to the security considerations for the it for 1884. Uh. Yeah, it's a little of storage fuck ups and then there's about twenty three hundred gas.
```
Comment (poojaranjan) [inline All Core Devs Meetings/meeting 94.md:217]: **Martin**

File: All Core Devs Meetings/meeting 94.md [hunk 8]
```diff
@@ -0,0 +1,437 @@
...
+  
+**Hudson** - Ok, I'll go ahead, Aleksi, just to finish it up
```
Comment (poojaranjan) [inline All Core Devs Meetings/meeting 94.md:318]: Ok, I'll go ahead, Alexey, just to finish it up

**Comment by Souptacular** [comment]
Thanks @alita-moore! Merging.

**Pull request merged**